data/bash_regression_test/test_run_script

```diff
@@ -0,0 +1,25 @@
+ #!/bin/bash
+ #count how many lines in regressions file, meaning how many tests failed.
+ repeat=$(awk 'END {print NR}' possible_regressions)
+ for (( i=1; i<=$repeat; i++)); do
```

Comment:
This for cycle reads lines from the file? Then the construction `while read line; do ...; done < possible_regressions` like in the *bash.pm* would be nicer.